<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="user_id_label">
      <property name="text">
       <string>用户ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_input"/>
    </item>
    <item>
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_input">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="user_type_label">
      <property name="text">
       <string>用户类型:</string>
      </property>
     </widget>
    </item>
    <!-- 这个部分需要修改 -->
    <item>
     <widget class="QWidget" name="user_type_widget">
      <layout class="QHBoxLayout" name="user_type_layout">
       <item>
        <widget class="QRadioButton" name="patient_radio">
         <property name="text">
          <string>病人</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="doctor_radio">
         <property name="text">
          <string>医生</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="admin_radio">
         <property name="text">
          <string>管理员</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="register_button">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
